### PR TITLE
Show big text for bluff/finesse

### PR DESCRIPTION
**Pull request by DarthGandalf**

### CODE REVIEW

File: docs/image-format.md [hunk 1]
```diff
@@ -23,6 +23,9 @@ import CardTextResult from '@site/image-generator/yml/generator-docs/card-text.y
 import TrashSource from '!raw-loader!@site/image-generator/yml/generator-docs/trash.yml';
 import TrashResult from '@site/image-generator/yml/generator-docs/trash.yml';
```
Comment (Zamiell) [inline docs/image-format.md:25]: no space between imports

Comment (DarthGandalf) [reply docs/image-format.md:25]: I disagree. Look above.
These imports are paired source/result

**Comment by Zamiell** [comment]
thanks

File: docs/image-format.md (hunk 1 above)
Comment (Zamiell) [reply docs/image-format.md:25]: ok, i take it back

**Pull request merged**